includes/handler/class-inbox.php

```diff
@@ -62,38 +58,24 @@ public static function maybe_handle_inbox_request( $data, $user_ids, $type, $act
 	 * @param string             $context      The context of the request (Inbox_Collection::CONTEXT_INBOX or Inbox_Collection::CONTEXT_SHARED_INBOX).
 	 */
 	public static function handle_inbox_requests( $data, $user_ids, $type, $activity, $context = Inbox_Collection::CONTEXT_INBOX ) {
-		$inbox_id = null;
-
 		/**
-		 * Filters the activity types to persist in the inbox.
+		 * Filter to defer inbox storage.
+		 *
+		 * Skip inbox storage for debugging purposes or to reduce load for
+		 * certain Activity-Types, like "Delete".
 		 *
-		 * @param array $activity_types The activity types to persist in the inbox.
+		 * @param bool  $defer Whether to defer inbox storage.
+		 * @param array $data  The activity data array.
+		 *
+		 * @return bool Whether to defer inbox storage.
 		 */
-		$activity_types = \apply_filters( 'activitypub_persist_inbox_activity_types', array( 'Create', 'Update', 'Follow', 'Like', 'Announce' ) );
-		$activity_types = \array_map( 'Activitypub\camel_to_snake_case', $activity_types );
+		$defer = \apply_filters( 'activitypub_defer_inbox_storage', false, $data );
```

Comment:
@obenland I am curious what you think about changing from opt-in to opt-out!

Reply:
I'm in favor!